lib/connect.js

```diff
@@ -224,18 +222,18 @@ function connect(token, options) {
   // 2 - Create the LocalParticipant using options.tracks.
   // 3 - Connect to rtc-room-service and create the RoomSignaling.
   // 4 - Create the Room and then resolve the CancelablePromise.
-  var cancelableRoomPromise = createCancelableRoomPromise(
+  const cancelableRoomPromise = createCancelableRoomPromise(
     getLocalTracks.bind(null, options),
     createLocalParticipant.bind(null, signaling, log, encodingParameters, options),
     createRoomSignaling.bind(null, token, options, signaling, iceServerSource, encodingParameters, preferredCodecs),
     createRoom.bind(null, options));
 
-  cancelableRoomPromise.then(function(room) {
+  cancelableRoomPromise.then(room => {
     log.info('Connected to Room:', room.toString());
     log.info('Room name:', room.name);
     log.debug('Room:', room);
     return room;
-  }, function(error) {
+  }, error => {
```

Comment:
@markandrus Isn't it safer to enforce braces around the arguments even if there is only one?

Reply:
What do you mean by safer? I don't know of any issue omitting braces. I can see that, if a function where to ever gain an additional argument, it's a slightly larger diff (because the parens have to be added), but otherwise I think it is OK.